src/gateways/createHighlightClient.ts

```diff
@@ -7,6 +9,8 @@ const formatError = (response: Response) => {
     });
 };
 
+export class DoNotHandleMe extends Error {}
```

Comment:
please rename this `UnauthenticatedError`, move it to a more generic location, and move the `dispatch(receiveLoggedOut())` to the middleware in app/utils